api-spec/protobuf/ark/v1/admin.proto

```diff
@@ -32,29 +30,23 @@ service AdminService {
   }
   rpc GetMarketHourConfig(GetMarketHourConfigRequest) returns (GetMarketHourConfigResponse) {
     option (google.api.http) = {
-      get: "/v1/admin/market-hour"
+      get: "/v1/admin/marketHour"
     };
   }
   rpc UpdateMarketHourConfig(UpdateMarketHourConfigRequest) returns (UpdateMarketHourConfigResponse) {
     option (google.api.http) = {
-      post: "/v1/admin/market-hour"
+      post: "/v1/admin/marketHour"
       body: "*"
     };
   }
-  rpc GetTxRequestQueue(GetTxRequestQueueRequest) returns (GetTxRequestQueueResponse) {
+  rpc ListIntents(ListIntentsRequest) returns (ListIntentsResponse) {
```

Comment:
we replaced txReq with intent, should we do it in other layers?(domain, app …)

Reply:
will do in a second pr